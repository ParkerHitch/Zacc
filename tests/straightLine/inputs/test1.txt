
print(1+(8+3*4)/4);

print(2*(-3+1));

