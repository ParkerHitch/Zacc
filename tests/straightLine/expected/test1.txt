PRINT: 6.0000
PRINT: -4.0000
